tests/kakuro/TestBoardCell.java

```diff
@@ -11,42 +13,52 @@
 
 public class TestBoardCell
 {
-    @Test
-    public void testNonInputCell()
-    {
-        // Arrange
-        BoardCell.CellType cellType = BoardCell.CellType.EMPTY;
-        // Act
-        BoardCell cell = new BoardCell(cellType);
-        // Assert
-        assertEquals(cell.getType(), BoardCell.CellType.EMPTY);
-    }
-
-    @Test
-    public void testInputCell()
-    {
-        // Arrange
-        int inputNumber = 1;
-        BoardCell.CellType cellType = BoardCell.CellType.EMPTY;
-        // Act
-        BoardCell cell = new BoardCell(cellType, inputNumber);
-        // Assert
-        assertEquals(cell.getFirstValue(), inputNumber);
-        assertEquals(cell.getType(), BoardCell.CellType.EMPTY);
-    }
-
-    @Test
-    public void testInputAnswerCell()
-    {
-        // Arrange
-        int inputNumber = 1;
-        int answerNumber = 3;
-        BoardCell.CellType cellType = BoardCell.CellType.EMPTY;
-        // Act
-        BoardCell cell = new BoardCell(cellType, inputNumber, answerNumber);
-        // Assert
-        assertEquals(cell.getFirstValue(), inputNumber);
-        assertEquals(cell.getSecondValue(), answerNumber);
-        assertEquals(cell.getType(), BoardCell.CellType.EMPTY);
-    }
+
+	/**
+	 *testInputCell is to check the BoardCell class and to check if those spaces where nothing are outputted are not causing an issue
+	 */
+	@Test
+	public void testNonInputCell()
+	{
+		// Arrange
+		BoardCell.CellType cellType = BoardCell.CellType.EMPTY;
```

Comment:
@Tiffany910 It is same code, it should be changed like that I suspect you used tabs and replaced spaces with tabs always use 4 tabs to ident everywhere instead of tabs because it shifts all the output and tags changes where there are none like here